jenkins/mellanox/buildimage-mlnx-all-rpc/Jenkinsfile

```diff
@@ -30,9 +30,10 @@ pipeline {
 
 git submodule foreach --recursive '[ -f .git ] && echo "gitdir: $(realpath --relative-to=. $(cut -d" " -f2 .git))" > .git'
 
+CACHE_OPTIONS="SONIC_DPKG_CACHE_METHOD=wcache SONIC_DPKG_CACHE_SOURCE=/nfs/dpkg_cache/mellanox"
```

Comment:
i do not quite understand why broadcom and mellanox are different cache directory, but mellanox and mellanox-rpc are the same cache directory.

Reply:
@xumia could you help answer this question?

Reply:
We only separate per platform, the Flag is added SONIC_COMMON_FLAGS_LIST to calculate the hash value.